input/pagecontent/reference.md

```diff
@@ -1544,25 +1651,103 @@ A defined rule set can be applied to an item by using an [`insert` rule](#rule-s
   * ^publisher = "Elbonian Medical Society"
   ```
 
-#### Defining Invariants
+#### Defining Value Sets
 
-Invariants are defined using the keywords `Invariant`, `Description`, `Expression`, `Severity`, and `XPath`. An invariant definition does not have any rules.
+A value set is a group of coded values, usually representing the acceptable values in a FHIR element whose data type is code, Coding, or CodeableConcept.
```

Comment:
Technically FHIR allows value sets to be applied to string elements as well.  But maybe that's why you said "usually"?

Reply:
How can FSH apply bindings to strings?

Reply:
I added string to the list of bindable types. But we should follow through with a FSH implementation of string binding.

Reply:
Does this mean a value set can be comprised of strings? How is that accomplished?

Reply:
The plot thickens.  In investigating this, I found that it also applies to the `uri` datatype.  Check it out here:
http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.binding

> Binds to a value set if this element is coded (code, Coding, CodeableConcept, Quantity), or the data types (string, uri).

Reply:
And here is further instruction:
http://hl7.org/fhir/R4/terminologies.html#strings

> In a few special cases, humans customarily use codes directly for elements that have type "string". A typical case is codes for states, and there are several places where a URI must come from a set of controlled values. An element of type string or uri can also be bound to a value set. When a string or URI is bound to a value set, the value property SHALL contain the code specified by the value set, and the system and display values are ignored.

Funny, we've used this as an example without even noticing that `state` is a `string`, not a `code`:
<img width="821" alt="image" src="https://user-images.githubusercontent.com/2278253/86918103-301d4800-c0f4-11ea-8184-da6b76e420f2.png">

Reply:
So it is unclear to me, since value sets contain Codings, does the display text or the code go into the string?

Reply:
(I wonder which one validates).

Reply:
I think this makes it pretty clear:
> When a string or URI is bound to a value set, the value property SHALL contain the code specified by the value set, and the system and display values are ignored.

If you're matching a `string` or `uri` against a value set, you're matching against the `code` (not the `display`).  We already support `<string> from {ValueSet}` and `<uri> from {ValueSet}` (_NOTE: `uri`, not `url`_):
<img width="632" alt="image" src="https://user-images.githubusercontent.com/2278253/86971442-2e2ca680-c13f-11ea-95ff-c916e65f3a80.png">

I don't think supporting it in assignment rules makes sense.  They should just assign it as a string.  It's supported for bindings because there is no other comparable way of doing it -- but assignments is possible (and even better) w/ just a string (not a code).

Reply:
Got it. Thanks!

Reply:
Fixed in documentation.